Portfolio/index.html

```diff
@@ -1,34 +1,147 @@
+        ex!<br><br>Name:Jhon Doe<br>Age:25<br>Exeperience:2years</p>
+    <h3>SKILLS</h3>
+    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT81jebNZhch3fhN8HiYmNkP76R6B38DzeH4g&usqp=CAU" height="20px"><br><span>html</span><br>
+    <img src="https://creazilla-store.fra1.digitaloceanspaces.com/icons/3247738/css3-alt-icon-md.png" height="20px"><br>
+    <span>css</span><br>
+    <img class="icon"
+        src="https://creazilla-store.fra1.digitaloceanspaces.com/icons/3204989/logo-javascript-icon-md.png"
+        height="20px" alt="JAVASCRIPT"><br><span>javascript</span><br>
+        <img class="logo" src="icons8-typescript-48.png" height="20px"><br><p>typesript</p>
+    <img class="logo" src="icons8-firebase-48.png" height="20px"><br><p>firebase</p>
+    <img class="logo" src="icons8-react-native-48.png"height="20px"><br><p>react</p>
 
+    <h3>PROJECTS</h3>
+    <div class="arr1">
+   <div class="screen"><img class="screen" src="JohnDeo.png" height="300px"></div> 
+   <div class="info"><h3>Taskbot Real-Time,Task Manager</h3>
+    <pr>React.js, Firebase<br>
+    <strong>Developed a dynamic task management application<br>
+            using React.js and Firebase<br>
+            User can create, update, and delete tasks in real-<br>
+            time across devices.<br>
+            Integrated Firebase Authentication for Secure user
+            access.
+    </strong>
+    </pr><br>
+    <button class="button">EXPLORE</button>
+</div>
+    </div>
+    <div class="arr2">
+    <div class="screen1">
+        <img src="https://cdn.jsdelivr.net/gh/UIX-Labs/uix-labs-Gulshan-Yadav-HTML-and-CSS-Crash-Course-Make-Your-own-Portfolio@a42854b83a5ee731356ced0e3cd79e4ac184f26d/public/screen2.png" height="300px">
+    </div>
+    <div class="info">
+        <h3>ShopPro, E-Commerce Website</h3>
+        React.js, TypeScript, Firebase<br>
+        <strong>Developed a responsive e-commerce website using<br>
+        Rect.js and TypeScript.<br>
+    Integrated Firebase for user authentication and<br>
+Firestore for product data storage.<br>
+Implemented secure payment processing with Strip API.</strong><br>
+<button class="button">EXPLORE</button>
+    </div>
+    </div> <br>
+    <div class="arr3">
+    <div class="screen2">
+        <img src="Screenshot 2023-11-10 072257.png" width="300px">
+    </div>
+    <div class="info2">
+        <h3>Eventzer,Event Planning & management</h3>
+        React Native,firebase <br>
+        <strong>Created a cross- platform mobile app using <br>
+        react native for planning. <br>
+        users can browse through and RSVP to <br>
+        event,view event details and receive real- <br>
+        time updates. <br>
+        utilized Firebase cloud Firestore for data <br>
+        storage and firebase cloud functions for push notification. <br>
+        <button class="button">EXPLORE</button>
+    </div>
+    </div>
+    <div class="arr4">
```

Comment:
The 'label' element has a spelling mistake. It should be 'label' instead of 'lable'.